tests/match_tracker/test_commands.py

```diff
@@ -300,5 +300,5 @@ def test_league_table_with_datetime_filter(self):
             )
 
         content = response["data"]["content"]
-        assert "global-user2 │      1 │        0 │   +2" in content
-        assert "global-user1 │      0 │        1 │   -2" in content
+        assert "global-user2 │      1 │        0" in content
```

Comment:
Might be worth adding some a class to do the functionality (using a dataclass/ attr.frozen?) that can _then_ be grabbed via the command class and spat into the `tabulate` string stuff.

That way, you could test the class specific properties and only assert new properties (if they are to be added) rather than have to update this string